printer.go

```diff
@@ -46,6 +46,9 @@ type Config struct {
 	// OmitPackagePaths, when true, causes the printer to omit the
 	// fully-qualified package path from the rendered type names.
 	OmitPackagePaths bool
+
+	// OmitUnexported omits unexported fields when set to true
+	OmitUnexported bool
```

Comment:
```suggestion
	// OmitUnexportedFields omits unexported struct fields when set to true.
	OmitUnexportedFields bool
```

> Could there be any other needed considerations for unexported things, other than struct fields?

We *could* omit exported fields that have unexported types, I suppose. That is possible but kind of rare and annoying so I don't consider it necessary at this point.  Nonetheless, I think it's worth making the name of the configuration field specifically mention fields.